src/Microsoft.ML.Tokenizers/EncodeSettings.cs

```diff
@@ -10,6 +10,7 @@ namespace Microsoft.ML.Tokenizers
     public struct EncodeSettings
     {
         public EncodeSettings() { MaxTokenCount = int.MaxValue; }
```

Comment:
could you please add the triple slashes doc for the constructor. Sorry, I missed that.